yeastphenome/apps/api/conditions.py

```diff
@@ -75,6 +77,19 @@ def get(self, request):
             queryset = conditions_query(query=None, taglist=taglist)
             count = len(queryset)
 
+        # Create field to sort phenotypes (2) and conditions (3)
+        if queryset:
+            queryset = queryset.annotate(
+                phenotype_list=GroupConcat(
+                    "condition__conditionset__dataset__phenotype__name", ", "
+                )
+            ).distinct()
+            queryset = queryset.annotate(
```

Comment:
It turns out the solution is much simpler. We don't even need GroupConcat -- Django as a series of postgres-specific aggregates: https://docs.djangoproject.com/en/3.1/ref/contrib/postgres/aggregates/#stringagg

In our case all we need to do is:
```
from django.contrib.postgres.aggregates.general import StringAgg
agg_field = 'condition__conditionset__dataset__paper__first_author'
queryset = queryset.annotate(paper_list=StringAgg(agg_field, delimiter=', ', distinct=True, ordering=agg_field))
```

Reply:
For each condition, this will make a string of unique first authors in alphabetical order.